tests/crc32/Makefile

```diff
@@ -41,5 +41,9 @@ warnp.o: ../../util/warnp.c ../../util/warnp.h
 test:	all
 	./test_crc32 -x
 
-perftest:	all
-	./test_crc32 -t
+perftest:
+	@${MAKE} all > /dev/null
```

Comment:
For certainty, this does not suppress output from cpusupport: that goes to stderr.  When running `perftest-generate.sh`, we do a `make all` before `perftest`, so this isn't a problem for the main "intended" way of generated perftest data".

I did consider capturing stderr as well, just to make sure that `make perftest` didn't have any undesired output, but I figured that if we had anything on stderr then it would be better to make sure it got displayed.